src/app/builder/builder.component.ts

```diff
@@ -92,6 +92,7 @@ export class BuilderComponent implements OnInit {
 
   onFeedUrlSubmit(value: string) {
     if (value) {
+      value = /^https?:\/\//i.test(value) ? value : `http://${value}`;
```

Comment:
So outside the scope of this ticket ... but if the feed url doesn't look like a url after this, can we show an error message, and not call router.navigate?  This would avoid the express proxy server throwing 500s, setting off our cloudwatch alarms, etc.

Pick your favorite url regexp!  (Something like `/(http|https):\/\/[\w-]+(\.[\w-]+)+([\w.,@?^=%&amp;:\/~+#-]*[\w@?^=%&amp;\/~+#-])?/`)